systemtest/src/test/java/io/strimzi/systemtest/AbstractST.java

```diff
@@ -110,16 +111,17 @@
     static final long TEARDOWN_GLOBAL_WAIT = 10000;
 
     public static KubeClusterResource cluster = new KubeClusterResource();
-    protected static DefaultKubernetesClient client = new DefaultKubernetesClient();
+    protected static DefaultOpenShiftClient client = new DefaultOpenShiftClient();
     static KubeClient<?> kubeClient = cluster.client();
 
     Resources resources;
+    static Resources clusterOperatorResources;
```

Comment:
Doing this we will have two `Resources` floating around. The cluster one and one used in each test. That kind of makes sense, I suppose since the one is "scoped" to the whole class and the other to the test method. But I wonder then if `clusterOperatorResources` is the best name. after it doesn't have to be just used for the CO. Would `testClassResources` be better?

Reply:
Good point, I will change it